### ISSUE TITLE
CTRL + N not working properly

### ISSUE THREAD

**Issue by RicaBenhossi** [Microsoft/vscode#54605]
The keyboard shortcut CTRL + N to create a new empty file only works when the focus is in the Welcome page, Side Bar, Settings file, Keyboard Shortcuts file or similar. If we hit the command in an empty file or an already open file, it doesn't work.

Open VsCode
Click at any point of t side bar
Hit ctrl+n
Click into the new file
Hit ctrl+n again (nothing happens)

**Comment by alexandrudima** [comment]
@RicaBenhossi Extensions can modify the default keybindings. What extensions do you have installed?

Regardless, the good news is that you can further modify them in any way you want. Can you open the keybindings editor `ctrl+k ctrl+s` and search in there for `ctrl+n`. What keybindings appear there? Can you add a screenshot? You can remove them if you don't want them.

**Comment by RicaBenhossi** [comment]
Hi Alexandru!

Tank
you for answering
so fast
and sorry for the delay in my response.

So. I tried to disable all extensions and reload the VSCode but the
problems weren't solved. So I went to my Keybindings.json and I don't know
why there was an empty shortcut using ctrl+N. Maybe some command previously
configurated there was removed and the key shortcut stoped working.

Anyway, after I remove this, the shortcut came back to normal.

Thanks again for your help and sorry about the mistake.
